packages/plugin-core/src/test/suite-integ/CreateDailyJournalNote.test.ts

```diff
@@ -1,21 +1,86 @@
 import * as vscode from "vscode";
 import { CreateDailyJournalCommand } from "../../commands/CreateDailyJournal";
-import { getActiveEditorBasename } from "../testUtils";
 import { expect } from "../testUtilsv2";
-import { runLegacyMultiWorkspaceTest, setupBeforeAfter } from "../testUtilsV3";
+import {
+  runLegacyMultiWorkspaceTest,
+  setupBeforeAfter,
+  withConfig,
+  EditorUtils,
+} from "../testUtilsV3";
+import { DVault } from "@dendronhq/common-all";
+import { sinon } from "@dendronhq/common-test-utils";
+import _ from "lodash";
+import { PickerUtilsV2 } from "../../components/lookup/utils";
+import { getActiveEditorBasename } from "../testUtils";
+
+const stubVaultPick = (vaults: DVault[]) => {
+  const vault = _.find(vaults, { fsPath: vaults[2].fsPath });
+  sinon.stub(PickerUtilsV2, "promptVault").returns(Promise.resolve(vault));
+  return vault;
+};
+
+suite("Daily Journal Suite", function () {
+  let ctx: vscode.ExtensionContext;
+  test("basic", (done) => {
+    runLegacyMultiWorkspaceTest({
+      ctx,
+      onInit: async ({}) => {
+        await new CreateDailyJournalCommand().run();
+        expect(
+          getActiveEditorBasename().startsWith("daily.journal")
+        ).toBeTruthy();
+        done();
+      },
+    });
+  });
+});
 
 suite("CreateDailyJournal", function () {
   let ctx: vscode.ExtensionContext;
 
   ctx = setupBeforeAfter(this, {});
 
-  test("basic", (done) => {
+  test("default journal vault", (done) => {
     runLegacyMultiWorkspaceTest({
       ctx,
-      onInit: async ({}) => {
+      onInit: async ({ wsRoot, vaults }) => {
+        withConfig(
+          (config) => {
+            config.lookupConfirmVaultOnCreate = false;
+            config.defaultDailyJournalVault = vaults[0].fsPath;
+            return config;
+          },
+          { wsRoot }
+        );
+        stubVaultPick(vaults);
         await new CreateDailyJournalCommand().run();
         expect(
-          getActiveEditorBasename().startsWith("daily.journal")
+          (await EditorUtils.getURIForActiveEditor()).fsPath.includes(
+            vaults[0].fsPath
+          )
+        ).toBeTruthy();
+        done();
+      },
+    });
+  });
+
+  test("default journal vault with lookup Confirm", (done) => {
+    runLegacyMultiWorkspaceTest({
+      ctx,
+      onInit: async ({ wsRoot, vaults }) => {
+        withConfig(
+          (config) => {
+            config.lookupConfirmVaultOnCreate = true;
+            config.defaultDailyJournalVault = vaults[0].fsPath;
+            return config;
+          },
+          { wsRoot }
+        );
+        await new CreateDailyJournalCommand().run();
```

Comment:
this should require `stubVaultPick`. the reason this is probably working is because we don't have a `sinon.restore` call. see https://github.com/dendronhq/dendron-site/blob/dev/vault/dendron.dev.qa.md#stubs